src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -31,13 +29,13 @@ public CinemaHallController(CinemaHallService cinemaHallService,
         this.cinemaHallDtoResponseMapper = cinemaHallDtoResponseMapper;
     }
 
-    @PostMapping
+    @PostMapping("/cinema-halls")
```

Comment:
you should move common part of endpoint to @RequestMapping annotation above class, the same for other classes